### PR TITLE
SO-2154 Support all reference set types in SNOMED CT request API

### PR DESCRIPTION
**Pull request by apeteri**

### CODE REVIEW

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/merge/MergeImpl.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2011-2016 B2i Healthcare Pte Ltd, http://b2i.sg
```
Comment (bulrich) [inline core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/merge/MergeImpl.java]: Kati Wolf-Joci Pápai

File: snomed/com.b2international.snowowl.snomed.api.impl/src/com/b2international/snowowl/snomed/api/impl/SnomedReleases.java [hunk 2]
```diff
@@ -33,19 +26,43 @@
...
+
+	/**
+	 * @return the code system representing the B2i extension of SNOMED CT
+	 */
+	public static CodeSystem b2iExtension(String extensionPath) {
 		return CodeSystem.builder()
-				.name(SNOMED_INT_NAME)
-				.shortName(SNOMED_INT_SHORT_NAME)
-				.oid(SNOMED_INT_OID)
+				.name("Systematized Nomenclature of Medicine Clinical Terms, B2i extension")
```
Comment (bulrich) [inline snomed/com.b2international.snowowl.snomed.api.impl/src/com/b2international/snowowl/snomed/api/impl/SnomedReleases.java]: Like CDO, SNOMED is no longer officially an acronym. I guess this should be **SNOMED CT, B2i extension**

File: snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedTerminologyComponentConstants.java [hunk 3]
```diff
@@ -58,8 +58,9 @@ private SnomedTerminologyComponentConstants() {}
 	public static final String DATA_TYPE_DATE = "com.b2international.snowowl.terminology.snomed.datatype.date";
 	public static final short DATA_TYPE_DATE_NUMBER = 109;
 	
-	public static final String SNOMED_INT_SHORT_NAME = "SNOMEDCT";
-	public static final String SNOMED_INT_NAME = "Systematized Nomenclature of Medicine Clinical Terms International Version";
+	public static final String SNOMED_SHORT_NAME = "SNOMEDCT";
+	public static final String SNOMED_NAME = "Systematized Nomenclature of Medicine Clinical Terms";
```
Comment (bulrich) [inline snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedTerminologyComponentConstants.java]: SNOMED CT

File: snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedTerminologyComponentConstants.java [hunk 4]
```diff
@@ -58,8 +58,9 @@ private SnomedTerminologyComponentConstants() {}
...
+	
 	public static final String SNOMED_INT_OID = "2.16.840.1.113883.6.96";
 	public static final String SNOMED_INT_LANGUAGE = "ENG";
 	public static final String SNOMED_INT_LINK = "http://www.ihtsdo.org";
```
Comment (bulrich) [inline snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedTerminologyComponentConstants.java]: This is now www.snomed.org

**Comment by apeteri** [comment]
Thanks for the review!

**Pull request merged**